Src/GoogleApis.Auth.DotNet4/OAuth2/GoogleCredential.cs

```diff
@@ -36,10 +32,18 @@ namespace Google.Apis.Auth.OAuth2
     /// See <see cref="GetApplicationDefaultAsync"/> for the credential retrieval logic.
     /// </para>
     /// </summary>
-    public abstract class GoogleCredential : IConfigurableHttpClientInitializer, ITokenAccess
+    public class GoogleCredential : IConfigurableHttpClientInitializer, ITokenAccess
     {
         private static DefaultCredentialProvider defaultCredentialProvider = new DefaultCredentialProvider();
```

Comment:
Consider adding comments for this field and for ICredential below as well.
Yeah I know it's private, but why not... :)